edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest/OptionAssertions.cs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest
+{
+    using System.Diagnostics.Contracts;
+    using FluentAssertions;
+    using FluentAssertions.Execution;
+    using FluentAssertions.Primitives;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public static class OptionAssertionExtensions
+    {
+        [Pure]
+        public static OptionAssertions<T> Should<T>(this Option<T> actualValue) => ShouldOption(actualValue);
+
+        [Pure]
+        public static OptionAssertions<T> ShouldOption<T>(this Option<T> actualValue)
```

Comment:
Yes, `Should()` is a common pattern with fluent assertions. But it doesn't work here because compiler selects `Should(object)` instead.
```csharp
public static ObjectAssertions Should(this object actualValue) 
  => new ObjectAssertions(actualValue);
``` 
I haven't spent much time on this though.

Reply:
I see. I vaguely recall running into a similar issue some time back...but I don't remember if/how I solved it.

Reply:
I found out that it is IDE that reported an error, not a compiler. Should be fine now